### PR TITLE
[Doc] Fix doc for wrong default value of expireTimeOfIncompleteChunkedMessage.

### PR DESCRIPTION
**Pull request by RobertIndie**
### Motivation

Currently, the doc for the default value of expireTimeOfIncompleteChunkedMessage is incorrect. The [documentation](https://pulsar.apache.org/api/client/2.9.0-SNAPSHOT/org/apache/pulsar/client/api/ConsumerBuilder.html#expireTimeOfIncompleteChunkedMessage-long-java.util.concurrent.TimeUnit-) says the default value is one hour, but it is actually one minute.

### Modifications

The original solution is to fix the incorrect default value in the code. But after the discussion, using one minute as the default value is a more appropriate way. So we decide to fix the doc and remain the original behavior in the code.

### Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

Need to update docs? 

- [x] `doc` 
  
Fix the doc to make it align with the code.

### CODE REVIEW

File: pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConsumerConfigurationData.java [hunk 1]
```diff
@@ -114,7 +114,7 @@ public int getMaxPendingChuckedMessage() {
 
     private boolean autoAckOldestChunkedMessageOnQueueFull = false;
 
-    private long expireTimeOfIncompleteChunkedMessageMillis = 60 * 1000;
+    private long expireTimeOfIncompleteChunkedMessageMillis = TimeUnit.HOURS.toMillis(1);
```
Comment (codelipenghui) [inline pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConsumerConfigurationData.java:117]: I think we should update the doc. 1 hour is too long for most cases. @merlimat @rdhabalia Please help confirm.

Comment (mattisonchao) [reply pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConsumerConfigurationData.java:117]: +1 .

Comment (RobertIndie) [reply pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConsumerConfigurationData.java:117]: +1 for updating the doc. If there are no other objections, I will update the doc.

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**